erdpy/projects/eei_registry.py

```diff
@@ -0,0 +1,262 @@
+            EEIFunction("managedDeployFromSourceContract", None, []),
+            EEIFunction("managedUpgradeContract", None, []),
+            EEIFunction("managedUpgradeFromSourceContract", None, []),
+            EEIFunction("managedAsyncCall", None, []),
+            EEIFunction("managedGetMultiESDTCallValue", None, []),
+            EEIFunction("managedGetESDTBalance", None, []),
+            EEIFunction("managedGetESDTTokenData", None, []),
+            EEIFunction("managedGetReturnData", None, []),
+            EEIFunction("managedGetPrevBlockRandomSeed", None, []),
+            EEIFunction("managedGetBlockRandomSeed", None, []),
+            EEIFunction("managedGetStateRootHash", None, []),
+            EEIFunction("managedGetOriginalTxHash", None, []),
+
+            # crypto
+            EEIFunction("sha256", None, []),
+            EEIFunction("managedSha256", useDifferentGasCostForReadingCachedStorageEpoch, []),
+            EEIFunction("keccak256", None, []),
+            EEIFunction("managedKeccak256", useDifferentGasCostForReadingCachedStorageEpoch, []),
+            EEIFunction("ripemd160", None, []),
+            EEIFunction("verifyBLS", None, []),
+            EEIFunction("verifyEd25519", None, []),
+            EEIFunction("verifySecp256k1", None, []),
+            EEIFunction("verifyCustomSecp256k1", None, []),
+            EEIFunction("encodeSecp256k1DerSignature", None, []),
+            EEIFunction("addEC", None, []),
+            EEIFunction("doubleEC", None, []),
+            EEIFunction("isOnCurveEC", None, []),
+            EEIFunction("scalarBaseMultEC", None, []),
+            EEIFunction("scalarMultEC", None, []),
+            EEIFunction("marshalEC", None, []),
+            EEIFunction("unmarshalEC", None, []),
+            EEIFunction("marshalCompressedEC", None, []),
+            EEIFunction("unmarshalCompressedEC", None, []),
+            EEIFunction("generateKeyEC", None, []),
+            EEIFunction("createEC", None, []),
+            EEIFunction("getCurveLengthEC", None, []),
+            EEIFunction("getPrivKeyByteLengthEC", None, []),
+            EEIFunction("ellipticCurveGetValues", None, [])
+        ]
+
+        self.functions_dict = {function.name: function for function in self.functions}
+
+    def sync_flags(self):
+        for flag in self.flags:
+            flag.sync(self.activation_knowledge)
+
+    def is_function_active(self, function_name: str) -> Union[bool, None]:
+        function = self.functions_dict.get(function_name, None)
+        if function is None:
+            return False
+        if function.activated_by is None:
+            return True
+        return function.activated_by.is_active
+
+
+class FeatureFlag:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.is_active: Union[bool, None] = None
+
+    def sync(self, knowledge: ActivationKnowledge):
+        self.is_active = knowledge.is_flag_active(self.name)
```

Comment:
This line also appears within the `try-except` block below.

Reply:
Good catch :partying_face:. Removed (it was by mistake).